pip/compat.py

```diff
@@ -41,7 +46,14 @@ def console_to_str(s):
         try:
             return s.decode(sys.__stdout__.encoding)
         except UnicodeDecodeError:
-            return s.decode('utf_8')
+            if WINDOWS:
+                try:
+                    from ctypes import cdll
+                    return s.decode("cp" + str(cdll.kernel32.GetACP()))
+                except ImportError:
```

Comment:
Good catch. As this code is Windows-only, ctypes should always be present (I think it can be omitted in certain types of Unix build, but it never will be on Windows).